### DISCUSSION TITLE
AWS 的 RDS 上执行 DDL 报错：Execute: Unexpected database port reported: 3306.

### DISCUSSION THREAD

**Discussion by ouyangxudu** [hhyo/Archery#1765] [Q&A]
执行类似 ALTER TABLE *** ADD *** DEFAULT '' NULL COMMENT 'abcd'  会在页面报错“Execute: Unexpected database port reported: 3306.”  
logs/archery.log 这个文件没有记录

**Comment by unknowissue** [comment]
看看后端日志有啥错误没？？
